### PR TITLE
Spelling: Home, in it, Bluetooth

### PR DESCRIPTION
**Pull request by comradekingu**

### CODE REVIEW

File: src/Permissions/Widgets/AppSettingsView.vala [hunk 1]
```diff
@@ -30,7 +30,7 @@ public class Permissions.Widgets.AppSettingsView : Gtk.Grid {
 
         var homefolder_widget = new PermissionSettingsWidget (
             Plug.permission_names["filesystems=home"],
-            _("Access your entire home folder, including any hidden folders."),
+            _("Access your entire Home folder, including any hidden folders in it."),
```
Comment (cassidyjames) [inline src/Permissions/Widgets/AppSettingsView.vala:33]: Not sure the last part is needed as it's implied, and sounds awkward to me when including it.

```suggestion
            _("Access your entire Home folder, including any hidden folders."),
```

File: src/Permissions/Widgets/AppSettingsView.vala [hunk 2]
```diff
@@ -58,7 +58,7 @@ public class Permissions.Widgets.AppSettingsView : Gtk.Grid {
 
         var bluetooth_widget = new PermissionSettingsWidget (
             Plug.permission_names["features=bluetooth"],
-            _("Manage bluetooth devices including pairing, unpairing, and discovery."),
+            _("Manage Bluetooth devices including pairing, unpairing, and discovery."),
```
Comment (cassidyjames) [inline src/Permissions/Widgets/AppSettingsView.vala:61]: :+1:

**Review by cassidyjames — CHANGES_REQUESTED** [review]
Generally it's better to propose these as their own PRs so unrelated changes aren't held up in case there is discussion about a different change.

For this specifically, the capitalization makes sense to me, but I don't think we need to add "in it" to the Home label; it's implied, and seems like unnecessary wordiness.

**Comment by jeremypw** [comment]
I agree that the words "in it" are superfluous after "including ..."

**Comment by jeremypw** [comment]
@comradekingu Are you able to update this PR?

**Review by danrabbit — APPROVED** [review]

**Pull request merged**